hw03_stubs.py

```diff
@@ -47,4 +47,3 @@ def main():
 
 main()  # Starts the program!
 
-
```

Comment:
There should be no edits to the hw03_stubs.py file! See Scott for help rolling it back.

Reply:
Looked at this in class. Hopefully it doesn't create too big of a problem...